client/src/pages/reports/Show.tsx

```diff
@@ -730,14 +742,13 @@ const ReportShow = ({ setSearchQuery, pageDispatchers }: ReportShowProps) => {
                     label="Attachments"
                     component={FieldHelper.ReadonlyField}
                     humanValue={
-                      <div className="attachment-card-list">
-                        {report.attachments.map(attachment => (
-                          <AttachmentCard
-                            key={attachment.uuid}
-                            attachment={attachment}
-                          />
-                        ))}
-                      </div>
+                      <AttachmentsDetailView
+                        attachments={attachments}
+                        updateAttachments={updateAttachments}
+                        relatedObjectType={Report.relatedObjectType}
+                        relatedObjectUuid={values.uuid}
+                        allowEdit={canEdit}
```

Comment:
This is not entirely correct; whether attachment edit is enabled also depends on `Settings.fields.attachment.restrictToAdmins` (see **reports/Form.tsx**).